Core/src/org/sleuthkit/autopsy/corecomponents/DataResultTopComponent.java

```diff
@@ -104,6 +109,8 @@ private void customizeComponent(boolean isMain, String title) {
 
         setTitle(title); // set the title
         setName(title);
+        getInputMap(JComponent.WHEN_ANCESTOR_OF_FOCUSED_COMPONENT).put(KeyStroke.getKeyStroke(KeyEvent.VK_B, InputEvent.CTRL_MASK), "addBookmarkTag");
```

Comment:
The user has no way to discover that this short cut exists. It should be added to the Tag File menu item, e.g., "Tag File (Ctrl + B)."

Also, this implementation allows you to tag things that do not have the "Tag File" context menu action and should not be able to be tagged. For example, the $Unalloc and $CarvedFiles virtual directories.